src/WorkbenchAccessManager.php

```diff
@@ -403,4 +433,39 @@ public function flushFields() {
     drupal_set_message($this->t('Field settings reset.'));
   }
 
+  /**
+   * Loads the saved role sections for a given role ID.
+   *
+   * @param string $role_id
+   *   The role ID.
+   *
+   * @return array
+   *   Sections for role.
+   */
+  protected function loadRoleSections($role_id) {
```

Comment:
Yeah, I recall someone asking 'Why did you use state for role section' to which the answer was 'because content entities' but if we had separate services for user section storage/retrieval and role section storage/retrieval, people could do what they wish in alternate implementations.

Not sure if you want to do that after this change, or as part of this change - let me know.